alembic/versions/034dcc59b63e_initial_revision.py

```diff
@@ -0,0 +1,110 @@
+    op.create_table('processed_files',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('url', sa.String(), nullable=False),
+                    sa.Column('create_date', sa.TIMESTAMP(
+                        timezone=True), nullable=False),
+                    sa.Column('update_date', sa.TIMESTAMP(
+                        timezone=True), nullable=False),
+                    sa.PrimaryKeyConstraint('id'),
+                    sa.UniqueConstraint('url')
+                    )
+    op.create_index(op.f('ix_processed_files_id'),
+                    'processed_files', ['id'], unique=False)
+    op.create_table('prediction_model_grid_subsets',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('prediction_model_id',
+                              sa.Integer(), nullable=False),
+                    sa.Column('geom', geoalchemy2.types.Geometry(geometry_type='POLYGON',
+                                                                 from_text='ST_GeomFromEWKT', name='geometry'), nullable=False),
+                    sa.ForeignKeyConstraint(['prediction_model_id'], [
+                        'prediction_models.id'], ),
+                    sa.PrimaryKeyConstraint('id'),
+                    sa.UniqueConstraint('prediction_model_id', 'geom')
+                    )
+    op.create_index(op.f('ix_prediction_model_grid_subsets_id'),
+                    'prediction_model_grid_subsets', ['id'], unique=False)
+    op.create_table('prediction_runs',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('prediction_model_id',
+                              sa.Integer(), nullable=False),
+                    sa.Column('prediction_run_timestamp', sa.TIMESTAMP(
+                        timezone=True), nullable=False),
+                    sa.ForeignKeyConstraint(['prediction_model_id'], [
+                        'prediction_models.id'], ),
+                    sa.PrimaryKeyConstraint('id'),
+                    sa.UniqueConstraint(
+                        'prediction_model_id', 'prediction_run_timestamp')
+                    )
+    op.create_index(op.f('ix_prediction_runs_id'),
+                    'prediction_runs', ['id'], unique=False)
+    op.create_table('predictions',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('prediction_run_id',
+                              sa.Integer(), nullable=False),
+                    sa.Column('prediction_model_grid_subset_id',
+                              sa.Integer(), nullable=False),
+                    sa.Column('prediction_timestamp', sa.TIMESTAMP(
+                        timezone=True), nullable=False),
+                    sa.Column('tmp_tgl_2', postgresql.ARRAY(
+                        sa.Float()), nullable=True),
+                    sa.Column('rh_tgl_2', postgresql.ARRAY(
```

Comment:
what do you think about adding comments for some of these columns? e.g., I don't know what 'tgl_2' stands for in 'tmp_tgl_2'

Reply:
I'd prefer not to add comments to auto-generated code - I guess I could, but really you'd need to refer to the actual model for context I think.